Userland/Libraries/LibWeb/Painting/StackingContext.cpp

```diff
@@ -146,61 +154,55 @@ void StackingContext::paint_internal(PaintContext& context) const
             paintable->after_children_paint(context, PaintPhase::Foreground, should_clip_overflow);
     };
 
+    // For a more elaborate description of the algorithm, see CSS 2.1 Appendix E
+    // Draw the background and borders for the context root (steps 1, 2)
```

Comment:
I think this comment should be at the top of the function still, that's where we usually put comments like this

Reply:
moved